### PR TITLE
Rr 18 fix spaces in resource capacity column in resource tab and code refactoring

### PR DESCRIPTION
**Pull request by RajatMCAF**
[IMP] I have refracted the code and changed resource_detail to resource

- changed the alignment for capacity

### CODE REVIEW

File: resource_reservation/__manifest__.py [hunk 1]
```diff
@@ -11,12 +11,16 @@
...
         'security/ir.model.access.csv',
+        'views/assets.xml',
         'views/resource_reservation_views.xml',
         'views/resource_reservation_tag_views.xml',
-        'views/resource_details_views.xml',
+        'views/resource_views.xml',
         'views/resource_availability.xml',
         'views/menu.xml'
     ],
+    'assets': {
```
Comment (aexitmcaf) [inline resource_reservation/__manifest__.py:21]: there are no file and no folders

File: resource_reservation/__manifest__.py [hunk 2]
```diff
@@ -11,12 +11,16 @@
     'depends': ['base'],
     'data': [
         'security/ir.model.access.csv',
+        'views/assets.xml',
```
Comment (aexitmcaf) [inline resource_reservation/__manifest__.py:14]: missing file

File: resource_reservation/__manifest__.py (hunk 1 above)
Comment (RajatMCAF) [reply resource_reservation/__manifest__.py:21]: I will remove it.

File: resource_reservation/__manifest__.py (hunk 2 above)
Comment (RajatMCAF) [reply resource_reservation/__manifest__.py:14]: I will remove it.

**Pull request closed without merging**